examples/x-to-zkevm-migration-app/frontend/tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
+    './src/components/**/*.{js,ts,jsx,tsx,mdx}',
+    './src/app/**/*.{js,ts,jsx,tsx,mdx}',
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+} 
```

Comment:
I believe the team is looking at using Biom3 for the examples now, Craig had recently done a pass through migrating the Tailwind setups over to Biom3. 

Would you be able to migrate the frontend over to Biom3 instead since we want to eventually render the examples in the docs site, so they should be using our branding?